### PR TITLE
fix: Fix select jump

### PR DESCRIPTION
**Pull request by istarkov**
## Description

1. What is this PR about (link the issue and add a short description)

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/design-system/src/components/select.stories.tsx [hunk 1]
```diff
@@ -86,26 +86,32 @@ export const WithDescriptions: StoryFn<typeof Select> = () => {
...
-      onChange={setValue}
-      getLabel={(option) => {
-        return option.label;
-      }}
-      getDescription={(option) => {
-        return option.description;
-      }}
-    />
+    <>
+      <div style={{ height: "300px" }}></div>
```
Comment (kof) [inline packages/design-system/src/components/select.stories.tsx:100]: lets add a comment what this is doing and wy specifically 300px, btw unit is opional

File: packages/design-system/src/components/select.stories.tsx [hunk 2]
```diff
@@ -86,26 +86,32 @@ export const WithDescriptions: StoryFn<typeof Select> = () => {
...
+        name="fruit"
+        options={options}
+        value={value}
+        getValue={(value) => value.label}
+        onChange={setValue}
+        getLabel={(option) => {
+          return option.label;
+        }}
+        getDescription={(option) => {
+          return <div style={{ width: "150px" }}>{option.description}</div>;
```
Comment (kof) [inline packages/design-system/src/components/select.stories.tsx:111]: in all other places we use description to define the width, why do we need it? why specifically 150? they can have different sizes

Comment (istarkov) [reply packages/design-system/src/components/select.stories.tsx:111]: It's storybook any value I like

**Comment by istarkov** [comment]
CLosing in favor of #3385

**Pull request closed without merging**